src/main/java/com/salesforce/dev/pages/Home/HomePage.java

```diff
@@ -12,59 +14,42 @@
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
+import static com.salesforce.dev.framework.CommonOperation.clickWebElement;
+
 /**
  * Created by Monica Pardo on 6/12/2015.
  */
 
- public class HomePage {
-
-   private WebDriver driver;
-   private WebDriverWait wait;
+public class HomePage {
 
-   @FindBy(id = "button-login")
-   @CacheLookup
-   WebElement loginBtn;
+    private WebDriver driver;
```

Comment:
Are you considering using an Abstract class to re-use the WebDriver and WebDriverWait?